Dynamic Programming/MirrorOfBinTree.py

```diff
@@ -0,0 +1,137 @@
+# https://practice.geeksforgeeks.org/problems/mirror-tree/1/
+
+
+
+'''
+class Node:
+    def _init_(self, val):
+        self.right = None
+        self.data = val
+        self.left = None
+'''
+# your task is to complete this function
+
+class Solution:
+    #Function to convert a binary tree into its mirror tree.
+    def createNode(self , val):
+        mirror = Node(0)
+        mirror.data = val
+        mirror.right = None
+        mirror.left = None
+        
+
+    def mirror(self,root):
+        
+        # Code here
+        if(root == None):
+            return
+        else:
+            temp = root
+            self.mirror(root.left)
+            self.mirror(root.right)
+            temp = root.left
+            root.left = root.right
+            root.right = temp
+            
+        return root
+        
+        
+
+#{ 
+#  Driver Code Starts
+#Initial Template for Python 3
+
+
+
+#Contributed by Sudarshan Sharma
```

Comment:
Why is it given in some other's name ? Please look into this.

Reply:
Hey so this question is solved on GFG , So we are just supposed to write the function part . The driver code is contributed by someone else. Ill edit and re submit

Reply:
Okay !